### ISSUE TITLE
User reported issue - nothing happens when entering first assessment date

### ISSUE THREAD

**Issue by AmaniKrayemRCPCH** [rcpch/rcpch-audit-engine#962]
"this is the step I get to, but when I click set nothing happens so I am unable to proceed to enter further information/data on the first paediatric assessment tabs etc. I have tried this now on multiple patients to no avail. "

![Image](https://github.com/rcpch/rcpch-audit-engine/assets/126471383/43a8b5e2-4b9c-4193-8755-3dc5856aa354)

She's confirmed that she's pressing the 'Set' button. I've asked her to confirm that she's getting the pop-up to confirm the date.

**Comment by eatyourpeas** [comment]
I have looked quite carefully through the logs and cant find anything. Could you possibly email me a little more detail to troubleshoot please?

**Comment by dc2007git** [comment]
I couldn't replicate this in my local live branch either. Will keep looking into it

**Comment by dc2007git** [comment]
Just some thoughts:

- It might be htmx-related. If a user logs into an account, goes to organisation summary, adds a child, then audits the child, then the first htmx they encounter is on that form. Is their JavaScript enabled on their browser?
- It might be worth clearing their cache, so the browser reloads everything fresh. This can be done with Shift + Refresh, but changes between systems, so could also be done with Ctrl + F5.
- @eatyourpeas The next step of the workflow is the Swal-2 container popping up asking the user for confirmation that they want to set an FPA date. Potentially the Swal-2 container isn't loading properly, so they can't progress because the window isn't loading for them.
- If none of this works, maybe switching browsers might?

**Comment by AmaniKrayemRCPCH** [comment]
She's confirmed that she doesn't get the pop-up to confirm the date
I'd already asked her to try a new browser, and she's tried both Chrome and Edge to no avail.

Will email you both with their details.

**Comment by eatyourpeas** [comment]
This is from your user's hospital in live, in the registration tab for one of the children registered there. The eligibility is correctly set and the organisation/site also correctly allocated. I don't think it can be a JS/htmx issue because the eligibility button uses that too. It is an interesting idea that the sweetalert plugin is in someway blocked by Macclesfield.  It is one of the few dependencies we are pulling in from a cdn. Perhaps we should serve this direct from disk? Hard to know without looking at your user's browser console. @mbarton any ideas?
<img width="1140" alt="image" src="https://github.com/rcpch/rcpch-audit-engine/assets/1058647/5b6e030a-2fab-4017-9838-afec0f9820e3">

**Comment by AmaniKrayemRCPCH** [comment]
Can I check if this has been merged into live @eatyourpeas ? So I can ask the user to try again

**Comment by eatyourpeas** [comment]
no it is not yet in live
there are a bunch of updates due to go over, but there are issues with the dev service currently which @mbarton and @pacharanero are thinking about this week

**Comment by eatyourpeas** [comment]
Hi @AmaniKrayemRCPCH - this has been in live since last week. Any update?

**Comment by AmaniKrayemRCPCH** [comment]
I don't have access to the E12 inbox anymore so @nikyraja will have to check if the user responded. 
I'll message you with their name, Niky.

**Comment by eatyourpeas** [comment]
No activity on this for some time, so hoping this is fixed. Do please reopen if I am wrong about this.
